crates/katana/core/src/service/messaging/starknet.rs

```diff
@@ -66,7 +66,7 @@ impl StarknetMessaging {
         from_block: BlockId,
         to_block: BlockId,
     ) -> Result<HashMap<u64, Vec<EmittedEvent>>> {
-        trace!(target: LOG_TARGET, "Fetching blocks {:?} - {:?}.", from_block, to_block);
+        trace!(target: LOG_TARGET, from_block = ?from_block, to_block = ?to_block, "Fetching blocks.");
```

Comment:
i think this should match ethereum logs 
```suggestion
        trace!(target: LOG_TARGET, from_block = ?from_block, to_block = ?to_block, "Fetching logs.");
```